scripts/report_generator.py

```diff
@@ -0,0 +1,58 @@
+import re
+import sys
+from datetime import datetime
+from pathlib import Path
+
+from utils.file_utils import ensure_directory_exists, write_file
+from utils.report_utils import create_report_config, create_report_sql
+from utils.system_utils import get_arg_value
+
+BASE_DIR = Path(__file__).parent.parent
+SQL_DIR = BASE_DIR / "models" / "reports" / "sql"
+CONFIG_DIR = BASE_DIR / "models" / "reports" / "config"
+
+
+def get_report_filename(name):
+    """Convert a report name to a valid filename."""
+    filename = name.lower().strip()
+    filename = re.sub(r"[\s_]+", "-", filename)
+    filename = re.sub(r"[^a-z0-9-]", "", filename)
+    filename = re.sub(r"-+", "-", filename)
+    return filename.strip("-")
+
+
+def main():
+    args = sys.argv[1:]
+    project = get_arg_value(args, "--project", "-p", "generic")
+    report_name = get_arg_value(
+        args, "--name", "-n", f"report-{datetime.now().strftime("%Y%m%d_%H%M%S")}"
+    )
```

Comment:
Nested double quotes inside f-string
```suggestion
    report_name = get_arg_value(
        args, "--name", "-n", f"report-{datetime.now().strftime('%Y%m%d_%H%M%S')}"
    )
```